metric/system/cgroup/reader_test.go

```diff
@@ -82,10 +125,18 @@ func TestReaderGetStatsV2(t *testing.T) {
 	require.NotNil(t, stats.Memory)
 	require.NotNil(t, stats.IO)
 
+	assert.Equal(t, pathv2, stats.Path)
+	assert.Equal(t, idv2, stats.ID)
```

Comment:
It seems the use of `assert` and `require` on this test is reversed. `require` will stop the test execution while `require` will keep going and logging all errors.

I believe the first two:
```go
	reader, err := NewReader(resolve.NewTestResolver("testdata/docker"), true)
	assert.NoError(t, err, "error in NewReader")

	stats, err := reader.GetV2StatsForProcess(312)
	assert.NoError(t, err, "error in GetV2StatsForProcess")
```
Should be `require` so the tests stops in case of failure, then we can just assert the rest.